tools/KoreBuild.Console/Commands/DockerFiles/winservercore.dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM microsoft/aspnet:4.6.2
+
+WORKDIR c:\\repo
+
+# DevPack returns exit 0 immediately, but it's not done, so we wait.
+# A more correct thing would be to block on a registry key existing or similar.
+RUN \
+        Invoke-WebRequest https://download.microsoft.com/download/F/1/D/F1DEB8DB-D277-4EF9-9F48-3A65D4D8F965/NDP461-DevPack-KB3105179-ENU.exe -OutFile .\\net461dev.exe ; \
```

Comment:
Hopefully this will be unnecessary once we fix https://github.com/aspnet/BuildTools/issues/399